lib/github/middleware.js

```diff
@@ -58,7 +82,10 @@ module.exports = function middleware (callback) {
       return
     }
 
-    const subscriptions = await Subscription.getAllForInstallation(gitHubInstallationId)
+    const subscriptions = await withNRSegment('middleware:Subscription.getAllForInstallation', async () => {
```

Comment:
Not really important: NewRelic names these slightly different: `Middleware: linkShared /slack/events.link_shared` should we follow this pattern of capital M and a space after the `:`? Just wondering if it's worth trying to be consistent here.

Reply:
Oh I definitely can. Good call!